packages/parse5-utils/src/index.js

```diff
@@ -24,16 +19,18 @@ const REGEXP_IS_HTML_DOCUMENT = /^\s*<(!doctype|html|head|body)\b/i;
  * @param {string} tagName Tag name of the element.
  * @param {Record<string, string>} attrs Attribute name-value pair array. Foreign attributes may contain `namespace` and `prefix` fields as well.
  * @param {string} namespaceURI  Namespace of the element.
+ * @returns {Element}
  */
 function createElement(tagName, attrs = {}, namespaceURI = DEFAULT_NAMESPACE) {
   const attrsArray = Object.entries(attrs).map(([name, value]) => ({ name, value }));
-  return adapter.createElement(tagName, namespaceURI, attrsArray);
+  return /** @type {Element} */ (adapter.createElement(tagName, namespaceURI, attrsArray));
```

Comment:
maybe this would be inferred now? maybe you don't need the type annotation anymore

Reply:
Removed.